spec/models/public_body_spec.rb

```diff
@@ -140,6 +140,19 @@
 
   end
 
+  describe '#expire_requests' do
+
+    it 'calls expire on all associated requests' do
+      authority = FactoryGirl.create(:public_body)
+      request = FactoryGirl.create(:info_request, :public_body => authority)
+      expect(authority).to receive(:info_requests).and_return([request])
```

Comment:
have fixed so the authority is now `request.public_body` but if I just `authority.reload` rather than mocking `info_requests`, I get the request, expire is called but my test doesn't see it happen